TestFoundation/TestNSStream.swift

```diff
@@ -33,7 +33,7 @@ class TestNSStream : XCTestCase {
     }
     
     func test_InputStreamWithData(){
-        let message: NSString = "Hello, playground"
+        let message: NSString = "Hello, playground\0\0\0"
```

Comment:
it would be better to write it's own tests instead of hijacking other un-related unit tests (this applies to all of the unit test changes here)